src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx

```diff
@@ -373,14 +373,14 @@ export const EvaluationProgressList = () => {
                   )}
               </div>
               {evaluatorToggledState[evaluatorIndex] && (
-                <table className='w-[90%] ml-14 mb-5 table-fixed whitespace-normal'>
+                <table className='w-9/10 ml-14 mb-5 table-fixed whitespace-normal'>
```

Comment:
I think these were not added in the tailwind config file?

Reply:
Donee :3